### PR TITLE
Polish README

### PR DESCRIPTION
**Pull request by agg505**
- Change tabs to spaces for better source code rendering
- Capitalize words for headings
- Unify usage for '-'

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@agg505 Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=26320)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-boot&pullRequestId=26320) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@agg505 Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=26320)!

File: README.adoc [hunk 1]
```diff
@@ -32,25 +32,25 @@ Here is a quick teaser of a complete Spring Boot application in Java:
...
 ----
 
 
 
-== Getting help
+== Getting Help
 Are you having trouble with Spring Boot? We want to help!
 
-* Check the {docs}/html/[reference documentation], especially the {docs}/html/howto.html#howto[How-to's] -- they provide solutions to the most common questions.
-* Learn the Spring basics -- Spring Boot builds on many other Spring projects; check the https://spring.io[spring.io] web-site for a wealth of reference documentation.
```
Comment (wilkinsona) [inline README.adoc:53]: I think this should be `--` so that they're rendered as an em dash.

File: README.adoc [hunk 2]
```diff
@@ -32,25 +32,25 @@ Here is a quick teaser of a complete Spring Boot application in Java:
...
+	    String home() {
+	        return "Hello World!";
+	    }
 
-		public static void main(String[] args) {
-			SpringApplication.run(Example.class, args);
-		}
+	    public static void main(String[] args) {
+	        SpringApplication.run(Example.class, args);
+	    }
```
Comment (wilkinsona) [inline README.adoc:43]: We prefer tabs as they're [more accessible](https://dev.to/alexandersandberg/why-we-should-default-to-tabs-instead-of-spaces-for-an-accessible-first-environment-101f).

**Review by wilkinsona — CHANGES_REQUESTED** [review]
Thanks very much for the pull request, @agg505. I left a couple of comments. Please let us know what you think.

File: README.adoc (hunk 2 above)
Comment (agg505) [reply README.adoc:43]: Ok, I see.

File: README.adoc (hunk 1 above)
Comment (agg505) [reply README.adoc:53]: So to be consistent, change the single ‘-‘ to double for line 56?

Comment (wilkinsona) [reply README.adoc:53]: Yes please.

**Comment by agg505** [comment]
Done. Have a check.

**Review by snicoll — APPROVED** [review]

**Comment by snicoll** [comment]
@agg505 thank you for making your first contribution to Spring Boot.

**Comment by agg505** [comment]
Such a great learning experience for my first open source contribution.
Thank you! @snicoll

**Pull request closed without merging**